caas/kubernetes/provider/k8s.go

```diff
@@ -1718,6 +1725,10 @@ func operatorConfigMapName(appName string) string {
 	return operatorName(appName) + "-config"
 }
 
+func operatorConfigurationsConfigMapName(appName string) string {
+	return deploymentName(appName) + "-configurations-config"
```

Comment:
wondering what the difference between `operatorConfigurationsConfigMapName` and `operatorConfigMapName`

Reply:
i think this gets created as part of the stateful set setup maybe? no sure, but it is there and is left behind so we ensure it is deleted